### ISSUE TITLE
Some usage instructions/code samples would be awesome!

### ISSUE THREAD

**Issue by adelevie** [boscomonkey/doi-rec-data#1]

**Comment by boscomonkey** [comment]
Good point. The code right now, is a proof of concept, and only demonstrates how to access one endpoint - "organizations". Code sample consists of lines #11-13 in rec_data.rb - https://github.com/boscomonkey/doi-rec-data/blob/master/rec_data.rb#L11-L13

A better README is needed. And perhaps turning this into a gem is useful too.

**Comment by adelevie** [comment]
:+1: to both :)
